.github/workflows/test.yml

```diff
@@ -0,0 +1,88 @@
+
+name: Test 
+
+on: 
+  push:
+    branches: 
+      - master
+  pull_request: 
+
+  workflow_dispatch:
+  
+
+
+jobs: 
+  test:
+    runs-on: ubuntu-20.04 
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: pip installs
+        run: pip install -U pytest requests
+
+      - name: Make temp directory 
+        run: mkdir /tmp/o2r
+
+      # Setup docker containers for testing
+      - name: Set up MongoDB
+        run: docker run --name mongodb -d -p 27017:27017 mongo:3.4
+
+      - name: Set up testloader
+        run: >-
+          docker run 
+          --name testloader  -d 
+          -p 8088:8088 
+          --link mongodb:mongodb 
+          -v /tmp/o2r:/tmp/o2r 
+          -v /var/run/docker.sock:/var/run/docker.sock 
+          -e LOADER_MONGODB=mongodb://mongodb:27017  
+          -e DEBUG=* 
+          o2rproject/o2r-loader:latest 
+
+      - name: Set up testmuncher
```

Comment:
I don't even find a reference to muncher anymore in the test code. Please comment out lines 50 to 60 and see if the tests still work.